### PR TITLE
Individual article

### PR DESCRIPTION
**Pull request by baloglu**
This PR:
- Adds the component to view artiles individually

### CODE REVIEW

File: src/components/ArticleId.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    const { article_id } = useParams();
+    const [article, setArticle] = useState({})
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(false)
+    const [isCommentsLoading, setIsCommentsLoading] = useState(false)
+    useEffect(() => {
+        setIsLoading(true)
+        fetch(`https://news-backend-vnab.onrender.com/api/articles/${article_id}`)
+            .then(result => {
```
Comment (MrMcSnail) [inline src/components/ArticleId.jsx]: You should extract all of your api calls to a utils/api.js module

File: src/components/Loading.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+function Loading() {
+    return (
+        <div className="loading-container">
```
Comment (MrMcSnail) [inline src/components/Loading.jsx:3]: NICE!

File: src/components/ArticleContainer.jsx [hunk 3]
```diff
@@ -44,7 +46,7 @@ function ArticleContainer(props) {
     )
     return (
         <>
-            {isLoading ? loading : content}
+            {isLoading ? <Loading /> : content}
```
Comment (MrMcSnail) [inline src/components/ArticleContainer.jsx:38]: No need for line 24 now right?

File: src/components/ArticleCard.jsx [hunk 4]
```diff
@@ -1,21 +1,20 @@
 import "../App.css"
+import { Link } from "react-router-dom";
+
 function ArticleCard({ article }) {
-    
+
     return (
-        <>
             <article className="card_container">
```
Comment (MrMcSnail) [inline src/components/ArticleCard.jsx]: I'd wrap your link around the whole card to make the clickable area more accessable

**Review by MrMcSnail — APPROVED** [review]
Looks good to Merge once you've read my comments

File: src/components/ArticleId.jsx (hunk 1 above)
Comment (baloglu) [reply src/components/ArticleId.jsx]: THanks.

**Pull request merged**